src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -36,25 +36,20 @@ With [our Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricin
 
-Note that our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus) provides up to 
-an additional 90 days of retention.
-
-Permission-related requirements for viewing and editing retention:
-* [Newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models): 
-  * Setting a contract with longer data retention for your entire organization requires the [<DoNotTranslate>**Billing**</DoNotTranslate> administration setting](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings)
-  * Adjusting retention within the bounds of a contract for specific types of data or for a specific account requires the [<DoNotTranslate>**Insights event retention**</DoNotTranslate> permission](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#data-retention) 
-* [Original user model](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model): requires the Owner role or Data Retention Manager Add-On role.
-
-## The data retention UI [#find-ui]
-
-The <DoNotTranslate>**Data retention**</DoNotTranslate> UI page is located in the [Data management hub](https://one.newrelic.com/data-management-hub). 
+* **Upgrade your retention plan (Max per contract)**: Your **Max per contract** column specifies the maximum number you can configure for the **Effective retention** of a data type. This maximum allowance always applies to all accounts in your organization if you have more than one account. To increase your **Max per contract**, you need a user who is in a group that has the **Billing** administrative setting enabled for it. This administrative setting applies to all users in that group. Those users will have a **Billing User** role assigned to them, but this method is the only way to assign that specific role. Only these users will see an **Upgrade plan** button in the top right of the **Data retention** UI.
+    <Callout variant="tip">
+      Some customers with upgraded retention plans can set **Effective retention** values to numbers higher than the corresponding **Max per contract** value (for an additional cost). If you need more details about this, please reach out to your New Relic account team or our [Sales team](https://newrelic.com/contact-us-customer).
+    </Callout>
+* **Change the current length of retention (Effective retention)**: This is the current retention time that you want a type of data to be stored and available to query. The highest number you can set this number to is in the corresponding **Max per contract** column. The users described below will be able to click the **Edit retention** button at the top right of the **Data retention** UI:
+    * For organizations with a single account: The same user described above who can upgrade the retention plan (**Max per contract**) can also set the effective retention values for the account.
```

Comment:
@rhetoric101 I find this bullet to be confusingly worded. If I'm reading it right we're just saying here that this works the same for single account organizations as it does for multi-account organizations.

Also, aren't most of our customers single accounts? Should we reorient this around single account customers and spell out the multi-account info as exceptions?